src/content/configuration/dev-server.md

```diff
@@ -136,7 +136,7 @@ module.exports = {
 To use this option with the CLI pass the `--allowed-hosts` option a comma-delimited string.
 
 ```bash
-webpack-dev-server --entry /entry/file --output-path /output/path --allowed-hosts .host.com,host2.com
+webpack serve --entry ./entry/file --output-path /output/path --allowed-hosts .host.com,host2.com
```

Comment:
```suggestion
webpack serve --entry ./entry/file --output-path ./output/path --allowed-hosts .host.com,host2.com
```
Can we use relative path for `--output-path` just like `--entry`? I think it's more reasonable as we're dealing with the current working directory.

Reply:
Fixed